test/README.md

```diff
@@ -28,8 +28,12 @@ All subsequent steps are executed within the test-fixture directory, `test/featu
 8. As EAS uses `yarn` as part of its initial processing we run `yarn import` independently to ensure a suitable `yarn.lock` file pre-exists, and so we have more control over the process
 9. To ensure peer-dependencies are installed we then invoke the `./run-bugsnag-expo-cli-install` script, which in turn calls the commands:
     - `bugsnag-expo-cli upload-sourcemaps` to add source-map upload hooks into the app configuration
-10. Finally, to ensure the apps are signed correctly for use on third-party testing services we copy a pre-written `credentials.json` into the repo, which is referenced in `eas.json`.
-    - More information about this file can be found within the `platforms-ansible` repository
+10. Finally, to ensure the apps are signed correctly for use on third-party testing services we copy a pre-written `credentials.json` and all associated credential files into the repo, which is referenced in `eas.json`.
+    - To run locally, these files will need to be imported from 1password (Expo p12 cert key v46+):
```

Comment:
Have we genericize this somehow?  I don't think we should talk about Bugsnag-specific tools in a public repo README.

Reply:
Ok, I've just added a section to refer to platforms-ansible for file details.  That should be enough without being explicit.